wkt/src/geometries.py

```diff
@@ -0,0 +1,50 @@
+from abc import ABC
+from enum import Enum
+
+from pydantic import BaseModel
+
+from traveltimepy import Coordinates
+from wkt.src.coordinates_models import (
```

Comment:
why do you need separate classes for coordinates?

Reply:
refactored code to avoid these separate classes